server/src/definitions/YmlDefinitionProvider.ts

```diff
@@ -62,34 +64,62 @@ export class YmlDefinitionProvider {
     }
 
     /**
-     * Add a definition to this provider.
+     * Add a definition to this provider. The element must have a definition location set.
```

Comment:
How about this? @mminot-yseop 
```suggestion
     * Add a definition to this provider. Elements without a definition location set will be ignored.
```

Reply:
> Elements without a definition location set will be ignored.

↓

> Elements without a set definition location will be ignored.

(“set” is both a noun and a verb, so let’s be careful xd I actually checked the code to see if it was a set, lol)

:heavy_plus_sign: do that for the other occurrence below.

Reply:
```suggestion
     * Add a definition to this provider. Elements without a set definition location will be ignored.
```